.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -36,9 +36,13 @@ Note: remove the backticks (``) from the link.
 - [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
 
 <!--
-If this release will become the next LTS release, add/update the checkboxes above as needed e.g.
+On the second Monday of each month, the current stable version will be promoted to the LTS release channel. In other words: 
+
+- If this release is promoted to stable on the first Monday of a given month, it needs to be promoted to LTS on the second Monday of the same month.
```

Comment:
What do we recommend if there is a cherry pick in stable during the first week of the month? Currently we wait ~3 days to determine whether a beta release is good to promote to stable. Does the new performance dashboard @samouri made change this answer?

Reply:
AFAIK there's no plan to deviate; the cherry-picked stable is still promoted to LTS the following week. @danielrozenberg would you suggest any alternative approach?

Reply:
I thought LTS comes from an older stable than one week?

Reply:
LTS is promoted from the current stable at promotion time, so it's a release that was **beta** for one week, then **stable** for one week, then **lts**